activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeUnAuthorizedTest.java

```diff
@@ -0,0 +1,301 @@
+
+import org.activiti.runtime.api.NotFoundException;
+import org.activiti.runtime.api.TaskAdminRuntime;
+import org.activiti.runtime.api.TaskRuntime;
+import org.activiti.runtime.api.model.Task;
+import org.activiti.runtime.api.model.builders.TaskPayloadBuilder;
+import org.activiti.runtime.api.query.Page;
+import org.activiti.runtime.api.query.Pageable;
+import org.activiti.runtime.api.security.SecurityManager;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.test.context.support.WithUserDetails;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@ContextConfiguration
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class TaskRuntimeUnAuthorizedTest {
+
+    @Autowired
+    private TaskRuntime taskRuntime;
+
+    @Autowired
+    private TaskAdminRuntime taskAdminRuntime;
+
+    private static String currentTaskId;
+
+    @Test
+    @WithUserDetails(value = "garth", userDetailsServiceBeanName = "myUserDetailsService")
+    public void aCreateStandaloneTaskForGroup() {
+
+        Task standAloneTask = taskRuntime.create(TaskPayloadBuilder.create()
+                .withName("group task")
+                .withGroup("doctor")
+                .build());
+
+        // the owner should be able to see the created task
+        Page<Task> tasks = taskRuntime.tasks(Pageable.of(0,
+                50));
+
+        assertThat(tasks.getContent()).hasSize(1);
+        Task task = tasks.getContent().get(0);
+
+        assertThat(task.getAssignee()).isNull();
+        assertThat(task.getStatus()).isEqualTo(Task.TaskStatus.CREATED);
+
+        currentTaskId = task.getId();
+
+
+    }
+
+    @Test(expected = NotFoundException.class)
+    @WithUserDetails(value = "salaboy", userDetailsServiceBeanName = "myUserDetailsService")
+    public void bClaimNotFoundBecauseYouAreNotACandidate(){
+        taskRuntime.claim(TaskPayloadBuilder.claim().withTaskId(currentTaskId).build());
+    }
+
+    @Test
+    @WithUserDetails(value = "admin", userDetailsServiceBeanName = "myUserDetailsService")
+    public void cCleanUpWithAdmin() {
+        Page<Task> tasks = taskAdminRuntime.tasks(Pageable.of(0, 50));
+        for (Task t : tasks.getContent()) {
+            taskAdminRuntime.delete(TaskPayloadBuilder
+                    .delete()
+                    .withTaskId(t.getId())
+                    .withReason("test clean up")
+                    .build());
+        }
+
+    }
+
+
+//
+//
+//    @Test
```

Comment:
What is the status of these tests?

Reply:
@erdemedeiros I think that I just forget to remove the comments because the tests should be covered in another class.

Reply:
If you search for: createStandaloneTaskReleaseUnAuthorized you will find the uncommented version of it.